ksql-engine/src/test/java/io/confluent/ksql/services/KafkaTopicClientImplTest.java

```diff
@@ -371,6 +372,39 @@ public void shouldAddTopicConfig() {
             defaultConfigEntry(TopicConfig.COMPRESSION_TYPE_CONFIG, "snappy")
         ));
 
+    expect(adminClient.incrementalAlterConfigs(
+        ImmutableMap.of(
+            new ConfigResource(ConfigResource.Type.TOPIC, "peter"),
+            ImmutableSet.of(
+              new AlterConfigOp(new ConfigEntry(TopicConfig.CLEANUP_POLICY_CONFIG, TopicConfig.CLEANUP_POLICY_COMPACT), AlterConfigOp.OpType.APPEND)
+        ))))
+        .andReturn(alterTopicConfigResponse());
+    replay(adminClient);
+
+    final KafkaTopicClient kafkaTopicClient = new KafkaTopicClientImpl(adminClient);
+    kafkaTopicClient.addTopicConfig("peter", overrides);
+
+    verify(adminClient);
+  }
+
+  @SuppressWarnings("deprecation")
+  @Test
+  public void shouldFallBackToAddTopicConfigForOlderBrokers() {
+    final Map<String, ?> overrides = ImmutableMap.of(
```

Comment:
While you are here, can you also add `Given:/When:/Then:` pattern to the tests :)

Reply:
Minimising changes... nope!